.github/workflows/windows.yml

```diff
@@ -142,69 +122,110 @@ jobs:
+            -DCMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }} `
             -S ${{ env.OPENVINO_REPO }} `
             -B ${{ env.BUILD_DIR }}
 
-      - name: Build
-        run: |
-          & {{ env.VCVARSPATH }} x64 && cmake --build ${{ env.BUILD_DIR }} --parallel ${{ steps.cpu-cores.outputs.count }} --config Release
-
-      - name: Install
-        run: cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/cmake_install.cmake
-
-      - name: Install Wheels
-        run: python3 -m pip install openvino-dev --find-links=${{ env.INSTALL_DIR }}\tools
-
-      - name: CMake Samples Tests
+      - name: Cmake build - OpenVINO
         run: |
-          & {{ env.VCVARSPATH }} x64 && cmake -S ${{ env.OPENVINO_REPO }}/tests/samples_tests -B ${{ env.BUILD_DIR }}/samples_tests
+          & {{ env.VCVARSPATH }} x64 && cmake --build ${{ env.BUILD_DIR }} --parallel --config ${{ env.CMAKE_BUILD_TYPE }}
 
-      - name: Install Samples Tests
-        run: cmake -DCOMPONENT=tests -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/samples_tests/cmake_install.cmake
-
-      - name: Install Tests
-        run: cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -DCOMPONENT=tests -P ${{ env.BUILD_DIR }}\cmake_install.cmake
-
-      - name: Cmake Layer Tests
+      - name: Cmake install - OpenVINO
         run: |
-          & {{ env.VCVARSPATH }} x64 && cmake -S ${{ env.OPENVINO_REPO }}/tests/layer_tests -B ${{ env.BUILD_DIR }}/layer_tests
-
-      - name: Build Layer Tests
-        run: cmake --build ${{ env.BUILD_DIR }}/layer_tests --parallel --config Release
-
-      - name: Install Layer Tests
-        run: cmake -DCOMPONENT=tests -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/layer_tests/cmake_install.cmake
+          cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/cmake_install.cmake
+          cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_TEST_DIR }} -DCOMPONENT=tests -P ${{ env.BUILD_DIR }}/cmake_install.cmake
+          cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -DCOMPONENT=python_wheels -P ${{ env.BUILD_DIR }}/cmake_install.cmake
 
       - name: Pack Artifacts
         run: |
-          $file=Get-ChildItem -Path "${{ env.INSTALL_DIR }}" -Exclude "tests"
+          $file=Get-ChildItem -Path "${{ env.INSTALL_DIR }}"
           $compress = @{
             Path = $file
             CompressionLevel = "Optimal"
             DestinationPath = "${{ env.BUILD_DIR }}/openvino_package.zip"
           }
           Compress-Archive @compress
 
-          $file=Get-ChildItem -Path "${{ env.INSTALL_DIR }}\tests"
+          $file=Get-ChildItem -Path "${{ env.INSTALL_TEST_DIR }}"
           $compress = @{
             Path = $file
             CompressionLevel = "Optimal"
             DestinationPath = "${{ env.BUILD_DIR }}/openvino_tests.zip"
           }
           Compress-Archive @compress
 
+      - name: Cmake & Build - OpenVINO Contrib
```

Comment:
for some reason whole OpenVINO starts to rebuild
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/7627f00b-72bd-4bdc-994e-a6403caa7e72)

Reply:
Yeah, I have no idea. Is something wrong with the CMake configuration commands between this and OV?

Reply:
Unfortunately, it did not help: https://github.com/openvinotoolkit/openvino/actions/runs/6457920465/job/17530516024. Seems like it is still building from the ground up.